src/misc/decorate.js

```diff
@@ -1,7 +1,8 @@
-goog.provide('ngeo.DecorateInteraction');
+goog.provide('ngeo.misc.decorate');
```

Comment:
nice! we should move the layer decorator here (#3309)

Reply:
Yes !